gcc.symbols.imp

```diff
@@ -75,154 +58,69 @@
-  { symbol: [ "pid_t", private, "<sys/msg.h>", public ] },
-  { symbol: [ "pid_t", private, "<sys/sem.h>", public ] },
-  { symbol: [ "pid_t", private, "<sys/shm.h>", public ] },
-  { symbol: [ "pid_t", private, "<sys/wait.h>", public ] },
-  { symbol: [ "pid_t", private, "<termios.h>", public ] },
-  { symbol: [ "pid_t", private, "<time.h>", public ] },
   { symbol: [ "pid_t", private, "<unistd.h>", public ] },
-  { symbol: [ "pid_t", private, "<utmpx.h>", public ] },
   { symbol: [ "ptrdiff_t", private, "<stddef.h>", public ] },
   { symbol: [ "regex_t", private, "<regex.h>", public ] },
   { symbol: [ "regmatch_t", private, "<regex.h>", public ] },
   { symbol: [ "regoff_t", private, "<regex.h>", public ] },
   { symbol: [ "sigevent", private, "<signal.h>", public ] },
-  { symbol: [ "sigevent", private, "<aio.h>", public ] },
-  { symbol: [ "sigevent", private, "<mqueue.h>", public ] },
-  { symbol: [ "sigevent", private, "<time.h>", public ] },
   { symbol: [ "siginfo_t", private, "<signal.h>", public ] },
-  { symbol: [ "siginfo_t", private, "<sys/wait.h>", public ] },
   { symbol: [ "sigset_t", private, "<signal.h>", public ] },
-  { symbol: [ "sigset_t", private, "<spawn.h>", public ] },
-  { symbol: [ "sigset_t", private, "<sys/select.h>", public ] },
   { symbol: [ "sigval", private, "<signal.h>", public ] },
   { symbol: [ "sockaddr", private, "<sys/socket.h>", public ] },
   { symbol: [ "socklen_t", private, "<sys/socket.h>", public ] },
-  { symbol: [ "socklen_t", private, "<netdb.h>", public ] },
   { symbol: [ "ssize_t", private, "<sys/types.h>", public ] },
-  { symbol: [ "ssize_t", private, "<aio.h>", public ] },
-  { symbol: [ "ssize_t", private, "<monetary.h>", public ] },
-  { symbol: [ "ssize_t", private, "<mqueue.h>", public ] },
-  { symbol: [ "ssize_t", private, "<stdio.h>", public ] },
-  { symbol: [ "ssize_t", private, "<sys/msg.h>", public ] },
-  { symbol: [ "ssize_t", private, "<sys/socket.h>", public ] },
-  { symbol: [ "ssize_t", private, "<sys/uio.h>", public ] },
   { symbol: [ "ssize_t", private, "<unistd.h>", public ] },
   { symbol: [ "stat", private, "<sys/stat.h>", public ] },
-  { symbol: [ "stat", private, "<ftw.h>", public ] },
   { symbol: [ "suseconds_t", private, "<sys/types.h>", public ] },
-  { symbol: [ "suseconds_t", private, "<sys/select.h>", public ] },
-  { symbol: [ "suseconds_t", private, "<sys/time.h>", public ] },
-  { symbol: [ "time_t", private, "<time.h>", public ] },
-  { symbol: [ "time_t", private, "<sched.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/msg.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/select.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/sem.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/shm.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/stat.h>", public ] },
-  { symbol: [ "time_t", private, "<sys/time.h>", public ] },
   { symbol: [ "time_t", private, "<sys/types.h>", public ] },
```

Comment:
I'm worrying that IWYU may start to suggest POSIX headers on non-POSIX-compliant systems. Specifying `--no_default_mappings` could be impractical when e.g. some project uses both Windows CRT as the C library and libc++ as the C++ library.

Reply:
I opened a PR for fixing that.

<https://github.com/include-what-you-use/include-what-you-use/pull/1288>